Source/WebCore/svg/SVGFontFaceUriElement.cpp

```diff
@@ -58,10 +59,7 @@ SVGFontFaceUriElement::~SVGFontFaceUriElement()
 
 Ref<CSSFontFaceSrcResourceValue> SVGFontFaceUriElement::createSrcValue() const
 {
-    ResolvedURL location;
-    location.specifiedURLString = getAttribute(SVGNames::hrefAttr, XLinkNames::hrefAttr);
-    if (!location.specifiedURLString.isNull())
-        location.resolvedURL = document().completeURL(location.specifiedURLString);
+    auto location = CSS::completeURL(getAttribute(SVGNames::hrefAttr, XLinkNames::hrefAttr), document()).value_or(CSS::URL::empty());
```

Comment:
Looks like here my annoyance at the name `url` caused me to use the word `location`, not `locator`, nor `srcURL`. 😂